Applications/DataExplorer/VtkVis/VtkRaster.cpp

```diff
@@ -54,15 +54,8 @@ vtkImageAlgorithm* VtkRaster::loadImage(const std::string &fileName,
 	{
 		raster = FileIO::AsciiRasterInterface::getRasterFromSurferFile(fileName);
 	}
-	if (raster) {
-		x0 = raster->getOrigin()[0];
-		y0 = raster->getOrigin()[1];
-		delta = raster->getRasterPixelSize();
-		double const*const data (raster->begin());
-		return VtkRaster::loadImageFromArray(data, x0, y0,
-						raster->getNCols(), raster->getNRows(), delta,
-						raster->getNoDataValue());
-	}
+	if (raster)
+		return VtkRaster::loadImageFromArray(raster->begin(), raster->getHeader());
```

Comment:
not part of this PR, i just did the syntactic changes for using RasterHeader.
i suppose i could change raster to a unique_ptr to solve the problem.